web/themes/custom/pineapple/sass/base/typography/_headings.scss

```diff
@@ -21,7 +21,7 @@ h1 {
 h2 {
   font-size: 30px;
   line-height: 42px;
-  font-weight: 600;
+  font-weight: 800;
```

Comment:
Why the increase in font size

Reply:
It was just a random design choice, but I reverted it.